x-pack/docs/en/security/securing-communications/tls-http.asciidoc

```diff
@@ -5,10 +5,29 @@
 When {security-features} are enabled, you can optionally use TLS to ensure that
 communication between HTTP clients and the cluster is encrypted.
 
-NOTE: Enabling TLS on the HTTP layer is strongly recommended but is not required.
-If you enable TLS on the HTTP layer in {es}, then you might need to make
-configuration changes in other parts of the {stack} and in any {es} clients that
-you use.
+IMPORTANT: You must enable TLS on the HTTP layer to use certain {es} features
+in a production environment, such as tokens and API keys.
+
+Enabling TLS on the HTTP layer is strongly recommended for non-production
+deployments, but is not required.
```

Comment:
I think if I were reading this, I would be confused as to whether it's required on production environments or not.

The sentence before says you only need it for certain features, but then this sentence say it's not required on non-production environments, but doesn't link back to those features.

Perhaps something more like
```
NOTE: Enabling TLS on the HTTP layer is strongly recommended but is not required.

IMPORTANT: In a production environment, some {es} features such as
tokens and API keys, will be disabled unless you enable TLS on the HTTP layer.
```

Also, I think `production environment` should probably link to https://www.elastic.co/guide/en/elasticsearch/reference/current/bootstrap-checks.html#dev-vs-prod-mode

Reply:
Thanks for the feedback @tvernum. I incorporated those changes into the [Encrypting HTTP client communications](https://elasticsearch_60858.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/7.8/configuring-tls.html#tls-http) page.